actions/terraform/init/action.yaml

```diff
@@ -90,13 +90,12 @@ runs:
       shell: bash
       if: always()
       working-directory: ${{ inputs.working_directory }}
-      env:
-        ARM_CLIENT_ID: ${{ inputs.arm_client_id }}
-        ARM_TENANT_ID: ${{ inputs.arm_tenant_id }}
-        ARM_SUBSCRIPTION_ID: ${{ inputs.tf_arm_subscription_id }}
-        ARM_USE_OIDC: "true"
       run: |
         terraform init -input=false \
+          -backend-config="use_oidc=true" \
+          -backend-config="use_azuread_auth=true" \
```

Comment:
This was the missing line, it's currently being added in the backend block in all pipelines.

Reply:
e.g. https://github.com/Altinn/altinn-platform/blob/main/infrastructure/adminservices-test/altinn-apim-test-rg/main.tf#L9
and 
https://github.com/Altinn/altinn-platform/blob/main/infrastructure/adminservices-test/altinn-monitor-test-rg/providers.tf#L9